src/lib/Liblog/chk_file_sec.c

```diff
@@ -228,7 +230,37 @@ tempstat(struct stat *sp, int isdir, int sticky, int disallow)
  *              			EACCESS if permissions are not ok
  */
 int
-chk_file_sec(char *path, int isdir, int sticky, int disallow, int fullpath)
+chk_file_sec(char *path, int isdir, int sticky, int disallow, int fullpath) {
+	return chk_file_sec_user(path, isdir, sticky, disallow, fullpath, 0);
+}
+
+
+/**
+ * @brief
+ * 	chk_file_sec_user() - Check file/directory security
+ *      Part of the PBS System Security "Feature"
+ *
+ * @par	To be secure, all directories (and final file) in path must be:
+ *		owned by userid < 10 or owned by uid if set
+ *		owned by groupid < 10 if group writable
+ *		not have world writable unless stick bit set & this is allowed.
```

Comment:
I'm not sure I understand this sentence.  The first part makes sense, but what about "& this is allowed"

It's also 'sticky bit', not 'stick bit'
(same below and above)

Reply:
I just copied this from chk_file_sec
For something to be allowed, it must not be in disallow.

Reply:
I still don't think I understand.  What do you mean by it must not be in disallow?

Reply:
If OtherWrite (S_IWOTH) is not passed into disallow
		 * Other write is allowed, and at least one of the following
		 * is true:
		 * - target is not a directory
		 * - target does not have sticky bit set
		 * - the value of the sticky argument we were passed was zero